.github/workflows/design-system-deploy.yml

```diff
@@ -0,0 +1,37 @@
+name: Deploy design.talend.com
+
+on:
+  push:
+#    paths:
+#      - 'packages/design-system/**'
```

Comment:
Not sure I understand why it's commented ^

Reply:
For testing purpose :D